expenses/schema.py

```diff
@@ -50,6 +50,12 @@ def resolve_can_edit(self, info):
     def resolve_can_cancel(self, info):
         return is_exec(info.context.user)
 
+    can_expense = graphene.Boolean()
+
+    def resolve_can_expense(self, info):
+        # User can expense if they are exec or a project lead and the status if one of these
```

Comment:
what do you mean by "can expense" exactly?

Reply:
Can expense determines whether the "Manage Status" section appears on the frontend. I wanted to keep the logic on the backend so in case in the future we change who has access to manage the status (ie approve reks, order items, etc.), it can be changed on the backend.